### PR TITLE
Second batch of layout fixes

### PR DESCRIPTION
**Pull request by heliocastro**

### CODE REVIEW

File: src/app/[locale]/preferences/components/NotificationSettings.tsx [hunk 1]
```diff
@@ -64,50 +61,32 @@ const NotificationSettings = ({ notificationSetting, setNotificationSetting }: P
...
-                            {Object.values(value.setting).map((entry) => (
-                                <div className='form-group' key={entry.id}>
-                                    <div className='form-check'>
-                                        <input
-                                            id={entry.id}
+                <>
+                    <Accordion.Item eventKey={value.documentType}>
+                        <Accordion.Header>{value.documentType}</Accordion.Header>
+                        <Accordion.Body>
+                            <Form name={value.documentType}>
```
Comment (hoangnt2) [inline src/app/[locale]/preferences/components/NotificationSettings.tsx:70]: Move this Form to div. This causes some errors
image

File: next.config.js [hunk 2]
```diff
@@ -11,7 +11,7 @@
 const withNextIntl = require('next-intl/plugin')()
 
 const nextConfig = {
-    reactStrictMode: true,
```
Comment (hoangnt2) [inline next.config.js:14]: I'm not sure we should get rid of react strict mode

File: src/app/[locale]/preferences/components/NotificationSettings.tsx (hunk 1 above)
Comment (heliocastro) [reply src/app/[locale]/preferences/components/NotificationSettings.tsx:70]: Fixed, as the entries were never returned before SSR is rendered.
No need for move back to div.

File: next.config.js (hunk 2 above)
Comment (heliocastro) [reply next.config.js]: Solved

File: src/app/[locale]/preferences/components/NotificationSettings.tsx (hunk 1 above)
Comment (heliocastro) [reply src/app/[locale]/preferences/components/NotificationSettings.tsx]: Fixed

**Review by hoangnt2 — APPROVED** [review]
Change look good

**Pull request merged**